src/sentry/integrations/utils/metrics.py

```diff
@@ -113,6 +113,10 @@ def add_extra(self, name: str, value: Any) -> None:
         """
         self._extra[name] = value
 
+    def add_extras(self, extras: Mapping[str, Any]) -> None:
```

Comment:
quick little utility to add multiple extras at the same time. i was using this initially in this pr, but dropped the usage - still figured it might be worth keeping around

Reply:
I've been thinking about this a bit. Do we want to make the typing on this stricter? Most of the time we're passing integers and strings around for logging only.
```suggestion
    def add_extras(self, extras: Mapping[str, int | str]) -> None:
```